parsers/browscap-php-3-full/parse

```diff
@@ -51,8 +51,8 @@ while (!$file->eof()) {
                         'version' => $r->platform_version,
                     ],
                     'device' => [
-                        'name'     => $r->device_name,
-                        'brand'    => $r->device_maker,
+                        'name'     => $r->device_code_name,
```

Comment:
This does change the results, did it seem like it better matched what other libs were looking at for these properties?

Reply:
fixed